.buildkite/pipeline.yaml

```
@@ -74,6 +74,7 @@ steps:
     key: "playwright-step"
     command: ".buildkite/scripts/run-playwright.sh"
     timeout_in_minutes: 15
+    branches: "!master"
```

```suggestion
    branches: "!(master|canary)"
```

Canary releases probably don't need to run Playwright either :)

---

True! chuck that canary in a cage ||🐦||